src/Impostor.Api/Games/IGame.cs

```diff
@@ -90,5 +90,24 @@ public interface IGame
         /// <param name="id">ID of the client.</param>
         /// <returns>A <see cref="ValueTask" /> representing the asynchronous operation.</returns>
         ValueTask SendToAsync(IMessageWriter writer, int id);
+
+        /// <summary>
+        ///     Creates a MessageWriter with GameData header.
+        /// </summary>
+        /// <remarks>Use with caution, recommended only for advanced developers.</remarks>
+        /// <param name="targetNetId">Net id of the InnerNetObject.</param>
+        /// <param name="callId">Rpc id of the message.</param>
+        /// <param name="targetClientId">Client id of GameDataTo's target.</param>
```

Comment:
maybe document what the value `-1` does

Reply:
Or just make it a nullable int?

Reply:
that's probably even better, but document it